README.md

```diff
@@ -236,33 +271,53 @@ As benchmarks depend on Ruby and Github-Linguist gem make sure you have:
  - Install it `gem install --no-rdoc --no-ri --local .linguist/github-linguist-*.gem`
 
 
-### How to reproduce current results
+#### Quick benchmark
+To run quicker benchmarks you can either:
+
+    make benchmarks
+
+to get average times for the main detection function and strategies for the whole samples set or:
+
+    make benchmarks-samples
 
+if you want to see measures per sample file.
+
+
+#### Full benchmark
 If you want to reproduce the same benchmarks as reported above:
  - Make sure all [dependencies](#benchmark-dependencies) are installed
  - Install [gnuplot](http://gnuplot.info) (in order to plot the histogram)
  - Run `ENRY_TEST_REPO="$PWD/.linguist" benchmarks/run.sh` (takes ~15h)
 
-It will run the benchmarks for enry and linguist, parse the output, create csv files and plot the histogram. This takes some time.
+It will run the benchmarks for enry and linguist, parse the output, create csv files and plot the histogram.
 
-### Quick
-To run quicker benchmarks you can either:
+### Faster regexp engine (optional)
 
-    make benchmarks
+[Oniguruma](https://github.com/kkos/oniguruma) is CRuby's regular expression engine.
+It is very fast and performs better than the one built into Go runtime. *enry* supports swapping
+between those two engines thanks to [rubex](https://github.com/moovweb/rubex) project.
+The typical overall speedup from using Oniguruma is 1.5-2x. However, it requires CGo and the external shared library.
+On macOS with brew, it is
 
-to get average times for the main detection function and strategies for the whole samples set or:
+```
+brew install oniguruma
+```
 
-    make benchmarks-samples
+On Ubuntu, it is
 
-if you want to see measures per sample file.
+```
```

Comment:
```suggestion
```bash
```

[optional] In several cases above we've used indentation for one-line code samples; it might be simpler to stick with that convention here too.

Reply:
Forcing markdown rendered with the infro-string to apply bash syntax highlighting does not change mach for single line. And batching lines makes a bit less sense here as each describes and alternative case.